a11y-input/test/ally-input.test.js

```diff
@@ -60,3 +62,16 @@ describe('a11y input', () => {
     `);
   });
 });
+
+/**
+ * Testing getter/setter
+ */
+describe('a11y input', () => {
+  it('can set/get the input value directly via the custom element', async () => {
+    const el = /** @type {A11yInput} */ (await fixture(html`
+      <a11y-input .value=${'foo'}></a11y-input>
+    `));
+
+    expect(el.value).to.equal('foo');
```

Comment:
#### Generates a `TypeError: Cannot set property 'value' of null at HTMLElement.set`

```
$ karma start --coverage

START:
Webpack bundling...
Hash: a525b6a4630559032483
Version: webpack 4.29.6
Time: 758ms
Built at: 03/18/2020 11:40:14 AM
                        Asset       Size                      Chunks             Chunk Names
ally-input.test.3651301707.js  354 bytes  ally-input.test.3651301707  [emitted]  ally-input.test.3651301707
                   commons.js   1.25 MiB                     commons  [emitted]  commons
     my-app.test.806894986.js  337 bytes       my-app.test.806894986  [emitted]  my-app.test.806894986
                   runtime.js   14.1 KiB                     runtime  [emitted]  runtime
Entrypoint ally-input.test.3651301707 = runtime.js commons.js ally-input.test.3651301707.js
Entrypoint my-app.test.806894986 = runtime.js commons.js my-app.test.806894986.js
18 03 2020 11:40:14.506:INFO [karma-server]: Karma v4.0.1 server started at http://0.0.0.0:9876/
18 03 2020 11:40:14.507:INFO [launcher]: Launching browsers ChromeHeadlessNoSandbox with concurrency unlimited
18 03 2020 11:40:14.512:INFO [launcher]: Starting browser ChromeHeadless
18 03 2020 11:40:14.886:INFO [HeadlessChrome 79.0.3945 (Mac OS X 10.13.6)]: Connected on socket Zbs2Lx9FczCwCgrTAAAA with id 9570611
18 03 2020 11:40:14.899:WARN [middleware:karma]: Invalid file type, defaulting to js. md
18 03 2020 11:40:14.899:WARN [middleware:karma]: Invalid file type, defaulting to js. md
  a11y input
    ✔ has a static Shadow DOM
    ✔ has 1 input and 1 label in the Light DOM
    ✖ can set/get the input value directly via the custom element
HeadlessChrome 79.0.3945 (Mac OS X 10.13.6) a11y input can set/get the input value directly via the custom element FAILED
	TypeError: Cannot set property 'value' of undefined
	    at HTMLElement.set value [as value] (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/a11y-input/src/ally-input.js:2:1)
	    at PropertyCommitter.commit (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/parts.js:353:1)
	    at PropertyPart.commit (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/parts.js:105:1)
	    at TemplateInstance.update (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/template-instance.js:40:1)
	    at NodePart._commitTemplateResult (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/parts.js:230:1)
	    at NodePart.commit (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/parts.js:172:1)
	    at render (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/render.js:44:1)
	    at litFixtureSync (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/@open-wc/testing-helpers/src/litFixture.js:14:1)
	    at litFixture (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/@open-wc/testing-helpers/src/litFixture.js:26:1)
	    at fixture (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/@open-wc/testing-helpers/src/fixture-no-side-effect.js:50:1)  My Filter App
    ✔ has a heading and a search field
    ✔ has by default an empty string as label

Finished in 0.268 secs / 0.013 secs @ 11:40:15 GMT-0500 (Central Daylight Time)

SUMMARY:
✔ 4 tests completed
✖ 1 test failed

FAILED TESTS:
  a11y input
    ✖ can set/get the input value directly via the custom element
      HeadlessChrome 79.0.3945 (Mac OS X 10.13.6)
    TypeError: Cannot set property 'value' of undefined        at HTMLElement.set value [as value] (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/a11y-input/src/ally-input.js:2:1)        at PropertyCommitter.commit (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/parts.js:353:1)        at PropertyPart.commit (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/parts.js:105:1)        at TemplateInstance.update (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/template-instance.js:40:1)        at NodePart._commitTemplateResult (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/parts.js:230:1)        at NodePart.commit (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/parts.js:172:1)        at render (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/lit-html/lib/render.js:44:1)        at litFixtureSync (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/@open-wc/testing-helpers/src/litFixture.js:14:1)        at litFixture (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/@open-wc/testing-helpers/src/litFixture.js:26:1)        at fixture (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/node_modules/@open-wc/testing-helpers/src/fixture-no-side-effect.js:50:1)
TOTAL: 1 FAILED, 4 SUCCESS
```

```js
=============================== Coverage summary ===============================
Statements   : 89.47% ( 17/19 )
Branches     : 50% ( 1/2 )
Functions    : 85.71% ( 6/7 )
Lines        : 89.47% ( 17/19 )
================================================================================
18 03 2020 11:40:15.396:ERROR [reporter.coverage-istanbul]: Coverage for statements (89.47%) does not meet global threshold (90%)
18 03 2020 11:40:15.396:ERROR [reporter.coverage-istanbul]: Coverage for lines (89.47%) does not meet global threshold (90%)
18 03 2020 11:40:15.396:ERROR [reporter.coverage-istanbul]: Coverage for branches (50%) does not meet global threshold (90%)
18 03 2020 11:40:15.396:ERROR [reporter.coverage-istanbul]: Coverage for functions (85.71%) does not meet global threshold (90%)
error Command failed with exit code 1.
```